src/goto-programs/interpreter.cpp

```diff
@@ -31,8 +41,41 @@ Function: interpretert::operator()
 
 void interpretert::operator()()
 {
+  show=true;
+  message->status() << "0- Initialize:" << messaget::eom;
+  initialise(true);
+  try
+  {
+    std::cout << "Type h for help" << std::endl;
+    while(!done) command();
+    message->status() << total_steps << "- Program End." 
+                      << messaget::endl << messaget::eom;
+  }
+  catch (const char *e)
+  {
+    message->error() << e << messaget::endl << messaget::eom;
+  }
+  while(!done) command();
+}
+
+/*******************************************************************
+
+Function: interpretert::initialise
+
+ Inputs:
+
+ Outputs:
+
+ Purpose: Initialises the memory map of the interpreter and 
+          [optionally] runs up to the entry point (thus doing 
+          non-program initialization)
+
+ \*******************************************************************/
+
+void interpretert::initialise(bool init) {
```

Comment:
Of course this pull request had been prepared well before the linter was available. Could you please run it across your pull request?